Tasks/NuGetPublisher/task.json

```diff
@@ -110,6 +110,15 @@
             "required": false,
             "helpMarkDown": "Optionally supply the path to NuGet.exe. Will override version selection.",
             "groupName": "advanced"
+        },
+        {
+            "name": "failOnEmptyNupkgMatch",
+            "type": "boolean",
+            "label": "Fail on empty nupkg match",
```

Comment:
I'd prefer a label of "Fail if no packages are found" or "Fail if no packages match the "Path/Pattern to nupkg"". The phrase "empty nupkg match" sounds too jargon-y to me.

You might also consider inverting the sense of the option and wording it as "Continue even if no packages are found" so that the option is "positive"

@alexmullans PM thoughts?